pkg/azure/ocpgwdeployer.go

```diff
@@ -96,7 +95,7 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 	}
 
 	gwNodeItems := gwNodes.Items
-	taggedExistingNodes := ocp.FilterDedicatedNode(gwNodeItems)
+	taggedExistingNodes := ocp.RemoveDuplicates(nil, gwNodeItems)
```

Comment:
why nil?

Reply:
It was an error while refactoring. Fixed it.